### PR TITLE
Update HAV ABI to underlying contract ABI

### PR DESCRIPTION
**Pull request by hav-noms**

### CODE REVIEW

File: contracts/contract-abi-eth.json [hunk 1]
```diff
@@ -271,7 +271,7 @@
...
...r","type":"address"},{"indexed":true,"name":"spender","type":"address"},{"indexed":false,"name":"value","type":"uint256"}],"name":"Approval","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"newTokenState","type":"address"}],"name":"TokenStateUpdated","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"proxyAddress","type":"address"}],"name":"ProxyUpdated","type":"event"},{"anonymous":false,"inputs":[],"name":"SelfDestructTerminated","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"beneficiary","type":"address"}],"name":"SelfDestructed","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"selfDestructDelay","type":"uint256"}],"name":"SelfDestructInitiated","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"newBeneficiary","type":"address"}],"name":"SelfDestructBeneficiaryUpdated","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"newOwner","type":"address"}],"name":"OwnerNominated","type":"event"},{"anonymous":false,"inputs":[{"indexed":false,"name":"oldOwner","type":"address"},{"indexed":false,"name":"newOwner","type":"address"}],"name":"OwnerChanged","type":"event"}]'
```
Comment (gamalielhere) [inline contracts/contract-abi-eth.json]: Is there any documentation to match this change? None of the functions seems to be listed from etherscan.

Comment (hav-noms) [reply contracts/contract-abi-eth.json]: So the 0xC011A72400E58ecD99Ee497CF89E3775d4bd732F with the existing ABI is the Havven Proxy contract. The actualy functions for the smart contract are listed in the underlying contract listed here as target https://etherscan.io/address/0xc011a72400e58ecd99ee497cf89e3775d4bd732f#readContract

Any functions called that are not local will be passed to the target contract using assembly.

See the target contract and this is where the ABI is from 
https://etherscan.io/address/0xd989a04ad891528b571ef73dacaefeb0402a65b3

for more info see here
https://github.com/Havven/havven/blob/master/contracts/Proxy.sol

Comment (gamalielhere) [reply contracts/contract-abi-eth.json]: Thanks for the clarification!

**Comment by hav-noms** [comment]
@gamalielhere do you have any idea of MEW or MC's release schedule for their web releases?

**Comment by gamalielhere** [comment]
@hav-noms it will probably be next week since we just released a new version this past Monday.

**Pull request merged**